### PR TITLE
DEMO activate review suggestions

### PR DESCRIPTION
**Pull request by daveatweaviate**
DEMO - delete later

[github howto](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)

### CODE REVIEW

File: developers/weaviate/some-broken-text.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Line one.
+Another line.
+
+## A heading
+
+More stuff.
+
+AnError
```
Comment (daveatweaviate) [inline developers/weaviate/some-broken-text.md:8]:
```suggestion
An Error
```

**Review by daveatweaviate — COMMENTED** [review]
@databyjp 

The suggest feature looks pretty straight forward. When you make a comment, the comment box has a little icon for suggesting changes near the comment's title bar.

It's the first icon, it looks like a doc page with a plus/minus sign.

I left another mistake on the last line if you want to try it out. Once you make the suggestion, I'll see what happens on my end as far as updating goes.

File: developers/weaviate/some-broken-text.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Line one.
+Another line.
+
+## A heading
+
+More stuff.
+
+Here re some ohr problems.
```
Comment (databyjp) [inline developers/weaviate/some-broken-text.md:8]:
```suggestion
Here be some other quandries.
```

**Comment by databyjp** [comment]
> @databyjp
> 
> The suggest feature looks pretty straight forward. When you make a comment, the comment box has a little icon for suggesting changes near the comment's title bar.
> 
> It's the first icon, it looks like a doc page with a plus/minus sign.
> 
> I left another mistake on the last line if you want to try it out. Once you make the suggestion, I'll see what happens on my end as far as updating goes.

Lgtm

File: developers/weaviate/some-broken-text.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+More stuff.
+
+Here be some other quandries.
+
+Morem, and more.
+
+An Error
+
+Anot her error.
```
Comment (daveatweaviate) [inline developers/weaviate/some-broken-text.md:14]:
```suggestion
Another error.
```

File: developers/weaviate/some-broken-text.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Line one.
+Another line.
+
+## A heading
+
+More stuff.
+
+Here be some other quandries.
+
+Morem, and more.
```
Comment (daveatweaviate) [inline developers/weaviate/some-broken-text.md:10]:
```suggestion
More, and more.
```

File: developers/weaviate/some-broken-text.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Line one.
```
Comment (daveatweaviate) [inline developers/weaviate/some-broken-text.md:1]:
```suggestion
Line one.

```

**Review by daveatweaviate — COMMENTED** [review]
Test for batch commits

**Comment by daveatweaviate** [comment]
Closing

**Pull request closed without merging**